ql/timeseries.hpp

```diff
@@ -182,90 +176,103 @@ namespace QuantLib {
         /*! \deprecated Use const_iterator instead and access the `first` data member.
                         Deprecated in version 1.31.
         */
-        [[deprecated("Use const_iterator instead and access the `first` data member.")]]
-        typedef boost::transform_iterator<projection_time, const_iterator> const_time_iterator;
+        [[deprecated(
+            "Use const_iterator instead and access the `first` data member.")]] typedef boost::
+            transform_iterator<projection_time, const_iterator>
+                const_time_iterator;
```

Comment:
This seems all jumbled together.  Do you happen to know how to configure clang-format so that it writes something more similar to the old code?

Reply:
Urgh. Okay, i admit defeat - i will update this PR to not include any reformatting.